conf/proxysql/config.sql

```diff
@@ -3,11 +3,12 @@ LOAD MYSQL VARIABLES TO RUNTIME;
 SAVE MYSQL VARIABLES TO DISK;
 
 DELETE FROM mysql_servers;
+INSERT INTO mysql_servers (hostgroup_id,hostname,port,max_replication_lag) VALUES (0,'mysql1',3306,1);
```

Comment:
This isn't needed, ProxySQL will set node with RW status to hostgroup 0 automatically